x-pack/plugins/reporting/server/export_types/png/create_job/index.ts

```diff
@@ -7,10 +7,11 @@
 import { cryptoFactory } from '../../../lib';
 import { CreateJobFn, CreateJobFnFactory } from '../../../types';
 import { validateUrls } from '../../common';
-import { JobParamsPNG } from '../types';
+import { JobParamsPNG, TaskPayloadPNG } from '../types';
 
 export const createJobFnFactory: CreateJobFnFactory<CreateJobFn<
-  JobParamsPNG
+  JobParamsPNG,
+  TaskPayloadPNG
```

Comment:
The CreateJobFn generics now specify the return type (job payload) of the create job function, which is a pretty big deal :)  

Before, it only specified the job params type.